### DISCUSSION TITLE
user-guide/create-bibliographies-within-zotero

### DISCUSSION THREAD

**Discussion by giscus** [zotero-chinese/wiki#260] [文档评论区]
# user-guide/create-bibliographies-within-zotero

Zotero 非官方中文维护小组

https://zotero-chinese.com/user-guide/create-bibliographies-within-zotero

**Comment by kan-yang13** [comment]
![image](https://github.com/user-attachments/assets/8829be97-c54b-4e5a-b52d-02c250b61ffa)
你好，我用这种方式导出的参考文献表粘贴到powerpoint或者word里都会出现编号在上一行最右边的情况，请问怎么解决？
